pkg/asset/installconfig/powervs/client.go

```diff
@@ -40,6 +42,8 @@ type API interface {
 	ListResourceGroups(ctx context.Context) (*resourcemanagerv2.ResourceGroupList, error)
 	ListServiceInstances(ctx context.Context) ([]string, error)
 	ServiceInstanceIDToCRN(ctx context.Context, id string) (string, error)
+	GetDatacenterCapabilities(ctx context.Context, region string) (map[string]bool, error)
```

Comment:
```suggestion
	GetDatacenterCapabilities(ctx context.Context, zone string) (map[string]bool, error)
```

Reply:
Good catch. Thanks. Lint doesn't catch that kind of things, eh? 🙂

Reply:
Addressed in the latest force-push. PTAL.